src/librustc_back/target/redox_base.rs

```diff
@@ -25,22 +25,14 @@ pub fn opts() -> TargetOptions {
             "-Wl,--as-needed".to_string(),
 
             // Always enable NX protection when it is available
-            "-Wl,-z,noexecstack".to_string(),
-
-            // Static link
-            "-static".to_string()
-        ],
-        late_link_args: vec![
-            "-lc".to_string(),
```

Comment:
We don't need these extra arguments anymore. -lc and -lm will be specified by the cross compiler